manim/renderer/shader_wrapper.py

```diff
@@ -0,0 +1,194 @@
+import os
+import re
+import moderngl
+import numpy as np
+import copy
+from .. import logger
+
+
+# from manimlib.utils.directories import get_shader_dir
+# from manimlib.utils.file_ops import find_file
+
+# Mobjects that should be rendered with
+# the same shader will be organized and
+# clumped together based on keeping track
+# of a dict holding all the relevant information
+# to that shader
+
+
+def get_shader_dir():
+    return "manim/renderer/shaders"
```

Comment:
```suggestion
    return Path(__file__).parent / "shaders"
```

Reply:
Perhaps, it's better to use [the first part of this suggestion ](https://github.com/ManimCommunity/manim/pull/926#discussion_r590446046) if multiple files (i.e. sounds and shaders) are going to require navigating the library.

Reply:
We do need to figure out our path handling globally eventually (i.e. soon) however, I don't think we should worry about it in this PR.

Reply:
I added this change manually